modules/task/backend.go

```diff
@@ -117,13 +96,45 @@ func (b *inMemBackend) Name() string {
 
 // Start implements the Module interface
 func (b *inMemBackend) Start(ready chan<- struct{}) <-chan error {
-	b.isRunning = true
-	return make(chan error)
+	errorCh := make(chan error, 2)
+	if b.IsRunning() {
+		errorCh <- errors.New("Cannot start when module is already running")
```

Comment:
style: errors should start with lowercase, they're meant to be interpolated into sentences